README.rst

```diff
@@ -318,6 +324,12 @@ Please do not report security issues in public. Send security concerns via email
 Changelog
 =========
 
+2.7.5 - 2021-02-16
```

Comment:
This date should be updated, right?  And the number should be 2.8.0, since this is a change in behavior.

Reply:
>This date should be updated, right?

Yes, I've missed that while copying the changelog entry.

>And the number should be 2.8.0, since this is a change in behavior.

Agreed :+1: 
I initially went for the minor version bump since the previous one already deployed this change, but you're right, a separate minor version is better.